apps/dashboard/src/hooks/use-api-keys-query.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+import { QueryKeys } from '@/utils/query-keys';
+import { useEnvironment } from '@/context/environment/hooks';
+import { IApiKey } from '@novu/shared';
+import { getApiKeys } from '../api/environments';
+
+export const useApiKeysQuery = () => {
```

Comment:
The pattern in the Dashboard so far is to name hooks as `use{Resource}` or `use{HTTPMethod}{Resource}`. So, based on that the hook should be converted to an API key retriever.

```suggestion
export const useApiKeys = () => {
```

Reply:
I looked at the only one that wasn't like this 😅 useTagsQuery. refactored.